lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1078,7 +1078,8 @@ static bool DeserializeAllCompilerFlags(swift::CompilerInvocation &invocation,
       swift::serialization::ExtendedValidationInfo extended_validation_info;
       info = swift::serialization::validateSerializedAST(
           buf, invocation.getSILOptions().EnableOSSAModules,
-          /*requiredSDK*/StringRef(), &extended_validation_info);
+          /*requiredSDK*/ StringRef(), /*requiresRevisionMatch*/ false,
```

Comment:
@adrian-prantl I assume `false` is what we want here

Reply:
Very good! Thanks